pkg/component/watch.go

```diff
@@ -50,26 +50,31 @@ func addRecursiveWatch(watcher *fsnotify.Watcher, path string, ignores []string)
 		}
 		return nil
 	})
+	fmt.Printf("\n\nIgnores is %s\n\n", ignores)
 	for _, v := range folders {
 		ignore := false
 		for _, pattern := range ignores {
-			if matched, _ := regexp.MatchString(pattern, v); matched {
+			// Don't watch:
+			// 	1. The main src directory as its files and sub-directories will already be watched
```

Comment:
Add comma after `The main src directory, as it's files...`

Reply:
Ack